.github/workflows/build-docker.yml

```diff
@@ -0,0 +1,38 @@
+name: Build Docker
+
+# Controls when the action will run. 
+# Triggers the workflow for a pull request for
+# dev when the Dockerfile file changes.
+on:
+  pull_request:
+    branches: [ dev ]
+    paths: [ Dockerfile ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
```

Comment:
supposed to be empty?

Reply:
I was was thinking this will let us set manually run this on the actions tab to test these GHA PRs. I removed it since I actually don't know much about this.